contributors/guide/pull-requests.md

```diff
@@ -590,6 +590,37 @@ at once to that file.
 * Can the file be improved further?
 * Does the trivial edit greatly improve the quality of the content?
 
+## Fixing linter issues
+
+Kubernetes has a set of linter checks. Some of those must pass in the entire
+code base, some must pass in new or modified code, and some are merely hints
+to developers how to improve their code.
+
+Please do not create Pull Requests for issues found by linters without first
+reaching out to maintainers on the `#code-organization`
+[Slack](http://slack.kubernetes.io) channel to determine whether there is
+sufficient interest in fixing such issues.
+
+It does not matter whether the linter is enabled in Kubernetes or not:
+- If a linter *is* enabled in
```

Comment:
```suggestion
- if the check performed by third party tools which are not integrated in
  Kubernetes CI or proprietary, file the bug or start a discussion about it first.
- If a linter *is* enabled in
```

Reply:
I want to acknoledge the fact that some people are running some third party analysis and not the standard linters. So we need to include it in scope

Reply:
Added at the end, with some small wording changes.